README.rst

```diff
@@ -98,21 +98,30 @@ Port  App                            Remarks
 8418  java-rabitmq-entry-point		 Reserved
 8419  java-rabitmq-propagator		 Reserved
 8420  java-rabitmq-sink				 Reserved
-====  =============================  ======== 
+====  =============================  ========
 
 Node.js applications
 ~~~~~~~~~~~~~~~~~~~~
 
 * HTTP access ports:
 
 ====  =============================
-Port  App                          
+Port  App
 ====  =============================
 5010  node-entry-point
 5011  node-propagator
 5012  node-sink
 ====  =============================
 
+MySQL
```

Comment:
Consider adding this before "Java Applications" table - all other "infrastructure" ports are there.

Reply:
Done